cli/actions/register_operator_with_avs.go

```diff
@@ -43,11 +51,15 @@ func RegisterOperatorWithAvs(ctx *cli.Context) error {
 		nodeConfig.EcdsaPrivateKeyStorePath,
 		ecdsaKeyPassword,
 	)
-	if err != nil {
-		return err
-	}
 
-	err = operator.RegisterOperatorWithAvs(operatorEcdsaPrivKey)
+	operator.SetAppointee(common.HexToAddress(nodeConfig.InstantSlasher), common.HexToAddress(nodeConfig.IncredibleSquaringServiceManager),common.HexToAddress(nodeConfig.AllocationManagerAddress),common.HexToAddress(nodeConfig.AVSRegistryCoordinatorAddress))
+	maxOperatorCount:=3
+	kickBpsOfOperatorStake:= 100
+	kickBpsOfTotalStake:= 1000
+	minimumStake :=0
+	multiplier:= 1
+	operator.CreateTotalDelegatedStakeQuorum(uint32(maxOperatorCount),uint16(kickBpsOfOperatorStake),uint16(kickBpsOfTotalStake),int64(minimumStake),int64(multiplier))
+	err = operator.RegisterForOperatorSets(common.HexToAddress(nodeConfig.AVSRegistryCoordinatorAddress),common.HexToAddress(nodeConfig.IncredibleSquaringServiceManager),operatorSetIds,waitForReceipt,*operator.BlsKeypair,socket,operatorEcdsaPrivKey)
```

Comment:
Can we format this?

Reply:
done https://github.com/Layr-Labs/incredible-squaring-avs/pull/120/commits/311744ff243c132e820578f59710d40d1e1c2075